### PR TITLE
chore: lint, proofread

### PR DESCRIPTION
**Pull request by paenis**
i left the actual text in journal and anonymity alone, just changed the md formatting. other stuff has grammar/other fixes

feel free to make any suggestions/changes!

### CODE REVIEW

**Comment by paenis** [comment]
i think this is basically ready

File: other/properly_sterilized.md [hunk 1]
```diff
@@ -90,32 +91,12 @@ Overall this is a very good method considering the other ones We've evaluated. I
 
 #TODO - [Guide is here](https://www.reddit.com/r/steroids/wiki/homebrew/list/#wiki_homebrew)
 
-## Manufactured by Tyger
```
Comment (hrtcat) [inline other/properly_sterilized.md:93]: looks like this section got dropped accidentally.

**Review by hrtcat — CHANGES_REQUESTED** [review]
This is amazing. Thank you thank you thank you. It's a much needed update. One minor issue I saw but once that's added back it's good to merge.

And also, thanks for getting this updated with the latest commit on main. I was just logging on to get it up to date myself!

File: other/properly_sterilized.md (hunk 1 above)
Comment (paenis) [reply other/properly_sterilized.md]: fixed!

**Review by hrtcat — APPROVED** [review]

**Comment by hrtcat** [comment]
Awesome! thanks for being our first contributor <3

**Pull request merged**